BUILD

```diff
@@ -0,0 +1,35 @@
+load("@rules_java//java:defs.bzl", "java_binary", "java_lite_proto_library", "java_proto_library")
```

Comment:
typically this BUILD file resides in the directory where the source code is (at least in g3 & blaze) -- any reason to put it here?

Reply:
+1, BUILD files reside in the directory of their files they are touching, it's the exception to see them like this.

Each package with any files that need to be compiled will generally contain one BUILD file, and it'll reference other BUILD files for dependencies, for example:

java_library(
  name = "gunshot_detection_trainer"
  deps = [
    "//java/com/google/location/lbs/activity/audioset/dataprocessing",
  ]
)

Reply:
Ok. So, I should place the build files in the same directory as the source code it references?